frontend/server/src/DAO/Contests.php

```diff
@@ -132,7 +136,7 @@ final public static function getByTitle(string $title) {
     final public static function getByAliasWithExtraInformation(string $alias): ?array {
         $sql = '
                 SELECT
```

Comment:
Acá también falta cerrar la cadena y concatenar
```suggestion
                SELECT' .
```